website/docs/r/alert_configuration.html.markdown

```diff
@@ -223,7 +239,8 @@ List of notifications to send when an alert condition is detected.
     - `WEBHOOK`
     - `MICROSOFT_TEAMS`
 
-* `notifier_id` - The notifier id is a system-generated unique identifier assigned to each notification method. This is needed when updating third-party notifications without requiring explicit authentication credentials.
+* `integration_id` - The ID of the associated integration, the credentials of which to use for requests.
```

Comment:
it's talking about credentials, is this a sensitive attribute?

Reply:
The id itself is not sensitive, is is being returned by the Atlas API. The id is used to reference an existing third party integration which configures credentials.

Reply:
ok, i would probably remove this part (or i don't understand it maybe): "the credentials of which to use for requests."

Reply:
I believe it is okay as it explains it uses credentials from the associated integration for requests. As this is how it is currently documented in the API docs prefer to leave as is.